benchmark/profile_generation.py

```diff
@@ -16,6 +16,8 @@
                     nvmlInit, nvmlShutdown, nvmlSystemGetDriverVersion)
 from tqdm import tqdm
 
+from lmdeploy import TurbomindEngineConfig
```

Comment:
could move this line into `profile_throughput` to avoid calling `get_logger`:
https://github.com/InternLM/lmdeploy/blob/a22ddab9fde114d5c4e03cd65bc0869b0f05d414/lmdeploy/messages.py#L12

https://github.com/InternLM/lmdeploy/blob/a22ddab9fde114d5c4e03cd65bc0869b0f05d414/lmdeploy/tokenizer.py#L12